src/controllers/customer.js

```diff
@@ -48,6 +50,28 @@ const createCustomer = async (req, res) => {
     }
 }
 
+const updateCustomer = async (req, res) => {
+  const id = Number(req.params.id)
+  const { name } = req.body
+
+  const customer = await prisma.customer.update({
+    where: {
+      id
+    },
+    data: {
+      name
+    },
+    include: {
+      contact: true
+    }
+  })
+
+  customer.contact = customer.contact[0]
```

Comment:
what is the purpose of this ? you had to add it because of the changed relation?

Reply:
I added it to satisfy the api spec, unlike the others this one expected a object, rather than an array of objects. Maybe it's because I changed the relationship